api.md

```diff
@@ -2986,7 +2986,7 @@ If the chunk's primary dimension is of a time datatype, `range_start` and
 | `range_end` | (TIMESTAMP WITH TIME ZONE) End of the range for the chunk's dimension |
 | `range_start_integer` | (BIGINT) Start of the range for the chunk's dimension, if the dimension type is integer based |
 | `range_end_integer` | (BIGINT) End of the range for the chunk's dimension, if the dimension type is integer based |
-| `is_compressed` | (BOOLEAN) Is the data in the chunk compressed?|
+| `is_compressed` | (BOOLEAN) Is the data in the chunk compressed? This is not applicable to chunks for distributed hypertables.|
```

Comment:
I find "not applicable" misleading, since this information is still applicable to distributed chunks, we just don't have a way to display it yet. Can we be more explicit and just say: "NULL will be displayed for distributed chunks. Use `chunk_compression_stats()` to get the compression status for distributed chunks."

Reply:
fixed